### PR TITLE
Sync develop changes March 25 - March 28 to hdf5_1_14

### PR DESCRIPTION
**Pull request by lrknox**

### CODE REVIEW

File: doxygen/dox/ExamplesAPI.dox [hunk 1]
```diff
@@ -433,8 +433,9 @@ FORTRAN
   MATLAB  PyHigh  PyLow
 </td>
 <td>h5ex_t_bit.h5</td>
-<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/112/h5ex_t_bit.tst">h5ex_t_bit.tst</a></td>
-<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/112/h5ex_t_bit.ddl">h5ex_t_bit.ddl</a></td>
+<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/16/h5ex_t_bit.tst">h5ex_t_bit.tst</a></td>
+<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/18/h5ex_t_bit.ddl">h5ex_t_bit.ddl</a></td>
+>>>>>>> 42f9d24... Correct reference copy for 16 API (#4244)
```
Comment (jhendersonHDF) [inline doxygen/dox/ExamplesAPI.dox]: Conflict line

File: doxygen/dox/ExamplesAPI.dox [hunk 2]
```diff
@@ -446,8 +447,9 @@ FORTRAN
   MATLAB  PyHigh  PyLow
 </td>
 <td>h5ex_t_cmpdatt.h5</td>
-<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/112/h5ex_t_cmpdatt.tst">h5ex_t_cmpdatt.tst</a></td>
-<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/112/h5ex_t_cmpdatt.ddl">h5ex_t_cmpdatt.ddl</a></td>
+<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/16/h5ex_t_cmpdatt.tst">h5ex_t_cmpdatt.tst</a></td>
+<td><a href="https://github.com/HDFGroup/hdf5/blob/hdf5_1_14/HDF5Examples/C/H5T/tfiles/18/h5ex_t_cmpdatt.ddl">h5ex_t_cmpdatt.ddl</a></td>
+>>>>>>> 42f9d24... Correct reference copy for 16 API (#4244)
```
Comment (jhendersonHDF) [inline doxygen/dox/ExamplesAPI.dox]: Conflict line

**Review by jhendersonHDF — APPROVED** [review]
Just the two minor comments to fix, but looked good to me

File: doxygen/dox/ExamplesAPI.dox (hunk 1 above)
Comment (lrknox) [inline doxygen/dox/ExamplesAPI.dox]:
```suggestion
```

**Review by byrnHDF — APPROVED** [review]

File: doxygen/dox/ExamplesAPI.dox (hunk 2 above)
Comment (lrknox) [inline doxygen/dox/ExamplesAPI.dox]:
```suggestion
```

**Pull request merged**